### ISSUE TITLE
Grading Overview

### ISSUE THREAD

**Issue by Lucav17** [kylepurpura/prj-rev-bwfs-dasmoto#1]
### Criteria 1: HTML and CSS Linking
* _Score Level:_ 4
* _Comment(s):_ Good job implementing the style sheet correctly! Implementing file paths is necessary for software development.

### Criteria 2: Implementation of Design Specification and Content
* _Score Level:_ 4
* _Comment(s):_ well done! The site looks good!
### Criteria 3: HTML Elements and Content
* _Score Level:_ 3
* _Comment(s):_ one thing I would add is that you should use more div tags in your code to show more sections rather than just one and to use more descriptive ID's

### Criteria 4: CSS Selectors and Syntax
* _Score Level:_ 4
* _Comment(s):_ Great job with your CSS!

### Overall Score: 15/16
Well done!

**Comment by kylepurpura** [comment]
Yep, agreed on the div tags. Slowly getting the hang of css. Thanks for the feedback. -K
